.github/workflows/static-analysis.yml

```diff
@@ -2,8 +2,8 @@ name: Static Analysis
 
 on:
   pull_request:
-    branches:
-      - '!release-plz-*'
+    branches-ignore:
+      - 'release-plz-*'

```

Comment:
Theoretically, the original should have worked, but didn't seem to be running. This seems to work now.